spec/trixer/slotter_spec.rb

```diff
@@ -173,19 +173,32 @@
         let(:booking4) { Slotter::Booking.new(id: 4, duration: 4, amount: 4, slot: 71) }
         let(:bookings) { [booking1, booking2, booking3, booking4] }
 
+        it { expect(subject[60]).to eql(capacity: 2, free_duration: { rest: 3 }) }
+        it { expect(subject[61]).to eql(capacity: 2, free_duration: { rest: 2 }) }
+        it { expect(subject[62]).to eql(capacity: 2, free_duration: { rest: 1 }) }
         it { expect(subject[64]).to eql(capacity: 2, booking: 1, free_duration: { rest: 0, 1 => 6 }) }
         it { expect(subject[65]).to eql(capacity: 2, booking: 1, free_duration: { rest: 0, 1 => 5 }) }
         it { expect(subject[66]).to eql(capacity: 2, booking: 1, free_duration: { rest: 0, 1 => 4 }) }
         it { expect(subject[67]).to eql(capacity: 2, booking: 1, free_duration: { rest: 0, 1 => 3 }) }
-        it { expect(subject[69]).to eql(capacity: 2, free_duration: { rest: 1 }) }
+        it { expect(subject[68]).to eql(capacity: 2, free_duration: { rest: 2, 2 => 5 }) }
+        it { expect(subject[69]).to eql(capacity: 2, free_duration: { rest: 1, 2 => 5 }) }
```

Comment:
it should increment for the current booking, so the expected value here would be `{ rest: 1, 2 => 6 }`

Reply:
at that point the max possible length is reached and to continue counting would mean to add code that is not really required.

Reply:
what if we want to add the feature to extend a booking to the left (start earlier). that would work only for one slot.